README.md

```diff
@@ -267,6 +267,7 @@ The [model_templates](model_templates) folder contains sample models that work w
 
 ##### Inference Models
 * [Scikit-Learn sample model](model_templates/inference/python3_sklearn)
+* [Scikit-Learn sample unsupervised anomaly detection model](model_templates/inference/python3_sklearn_anomaly)
```

Comment:
I thought as a different section:
- inference
- training
- anomaly

but up to you

Reply:
I disagree

Reply:
why

Reply:
anomaly models aren't a third category, they fit the same way as multiclass within the inference training dichotomy.

Reply:
you are right